platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/junit/OtherTestExtension.java

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: Apache-2.0
+package org.hiero.otter.fixtures.junit;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.lang.reflect.Parameter;
+import java.time.Duration;
+import java.util.Objects;
+import java.util.Optional;
+import org.hiero.otter.fixtures.TestEnvironment;
+import org.hiero.otter.fixtures.turtle.TurtleTestEnvironment;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
+import org.junit.jupiter.api.extension.InvocationInterceptor;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+import org.junit.jupiter.api.extension.ParameterResolver;
+import org.junit.jupiter.api.extension.TestWatcher;
+
+public class OtherTestExtension implements InvocationInterceptor, ParameterResolver, TestWatcher {
```

Comment:
Is this supposed to be `OtterTestExtension`?

Reply:
lol, thanks :)